python/seldon_core/microservice.py

```diff
@@ -353,6 +353,102 @@ def parse_args():
     return parser.parse_known_args()
 
 
+def _make_rest_server_debug(
+    user_object: Any,
+    seldon_metrics: SeldonMetrics,
+    args: argparse.Namespace,
+    *,
```

Comment:
Here there is an expected number of args and kwargs so no need to allow anonymous args

Reply:
this is not about annoymous args, but keyword-only args, https://peps.python.org/pep-3102/

Reply:
but I can remove `*`, if it's considered unnecessary.

Reply:
Yeah, anonimous args as allowing for any further args, but not sure why this is needed so best remove